### PR TITLE
Update the user-agents

### LINKED ISSUE #14: Update UserAgent headers to accommodate for version filter
See https://github.com/mozilla-services/contile/pull/247

### PR DESCRIPTION
**Pull request by ashrivastava-qa**
## Description

Updated the user agents for the tests.

## Issue(s)

Closes #14

### CODE REVIEW

**Review by hackebrot — COMMENTED** [review]
Hi @ashrivastava-qa! 👋🏻 

Did you verify these new `User-Agent` header values to see if Contile handles them as expected?

**Comment by ashrivastava-qa** [comment]
Hi @hackebrot, I tested these header values in the Woothee's demo site : `http://woothee.github.io/` and Contile uses the same library (Woothee) for parsing the User-Agent Header. So Contile will be able to handle them.

File: loadtest.py [hunk 1]
```diff
@@ -9,24 +9,22 @@
...
     'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:10.0) Gecko/20100101 '
-    'Firefox/10.0',
+    'Firefox/91.0',
 
-    'Mozilla/5.0 (X11; Linux x86_64; rv:10.0) Gecko/20100101 Firefox/10.0',
-    'Mozilla/5.0 (Android; Mobile; rv:40.0) Gecko/40.0 Firefox/40.0',
-    'Mozilla/5.0 (Android; Tablet; rv:40.0) Gecko/40.0 Firefox/40.0',
-    'Mozilla/5.0 (Android 4.4; Mobile; rv:41.0) Gecko/41.0 Firefox/41.0',
-    'Mozilla/5.0 (Android 4.4; Tablet; rv:41.0) Gecko/41.0 Firefox/41.0',
+    'Mozilla/5.0 (X11; Linux x86_64; rv:90.0) Gecko/20100101 Firefox/91.0',
```
Comment (hackebrot) [inline loadtest.py]: I suggest we have at least one `User-Agent` for Firefox 90 to test for legacy image handling:

```suggestion
    'Mozilla/5.0 (X11; Linux x86_64; rv:90.0) Gecko/20100101 Firefox/90.0',
```

**Review by hackebrot — CHANGES_REQUESTED** [review]

**Comment by ashrivastava-qa** [comment]
Yes that's a good idea, committed the suggestion. Thanks!

**Review by hackebrot — APPROVED** [review]
LGTM! Thanks @ashrivastava-qa! 👍🏻

**Comment by ashrivastava-qa** [comment]
@pjenvey I merged the PR before seeing that Raphael requested a review from you. I'd very much like for you to review it still and let me know if the changes looks good to you. Thanks!

**Pull request merged**